src/components/shared/MarkdownContainer.astro

```diff
@@ -64,10 +64,11 @@ const { class: className } = Astro.props;
             @apply mb-4 w-full rounded-xl;
         }
         & :global(pre) {
-            @apply mb-8 rounded text-sm !bg-code border-2 border-border relative;
+            @apply mb-8 rounded text-sm !bg-code border-[1px] border-code relative;
+            line-height:1.40rem;
```

Comment:
@kearfy is this the correct way of doing line height?

Reply:
Should be `leading-[1.4rem]`